examples/pytorch/rgat/rgat.py

```diff
@@ -58,12 +56,9 @@ def forward(self, blocks, x):
 
 split_idx = dataset.get_idx_split()
 train_idx, valid_idx, test_idx = split_idx['train'], split_idx['valid'], split_idx['test']
-
-if not USE_UVA:
-    graph = graph.to('cuda')
-    train_idx = recursive_apply(train_idx, lambda x: x.to('cuda'))
-    valid_idx = recursive_apply(valid_idx, lambda x: x.to('cuda'))
-    test_idx = recursive_apply(test_idx, lambda x: x.to('cuda'))
+train_idx = recursive_apply(train_idx, lambda x: x.to('cuda'))
```

Comment:
How about renaming the function to `hetero_apply`?

Reply:
Hmm I think "hetero apply" is vaguer and a bit too specific?  Like "recursive_apply" can work on a dict of node features, a dict of edge features, and a dict of dict of something, etc.